components/IssueCredentialOptions.yml

```diff
@@ -1,44 +1,47 @@
-            type:
-              type: string
-              description: The type of credential status to issue the credential with
-        mandatoryPointers:
-          type: array
-          description: Used with selective disclosure schemes to specify mandatory reveal statements
-      example:
-        {
-          "created": "2020-04-02T18:48:36Z",
-          "credentialStatus": { "type": "RevocationList2020Status" },
-        }
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  schemas:
+    IssueCredentialOptions:
+      type: object
+      additionalProperties: false
+      description: Options for specifying how the DataIntegrityProof is created.
+      properties:
+        created:
+          type: string
+          description: The date and time of the proof (with a maximum accuracy in seconds). Default current system time.
+        challenge:
+          type: string
+          description: A challenge provided by the party requesting the proof. For example, 6e62f66e-67de-11eb-b490-ef3eeefa55f2
+        domain:
+          type: string
+          description: The intended domain of validity for the proof. For example, website.example
+
+        credentialStatus:
+          type: object
+          description: The method of credential status to include in the issued credential. If omitted, credential status will be included.
+          properties:
+            type:
+              type: string
+              description: The type of credential status to issue the credential with
+        mandatoryPointers:
+          type: array
+          description: Used with selective disclosure schemes to specify mandatory reveal statements
+        credentialId:
+          type: string
+          description: A URI that can be used to identify the credential, that can be used in APIs to refer to the issued verifiable credential. If credentialId is not provided by the issuer coordinator, the issuer service will auto-populate its value from credential.id. If neither credentialId nor credential.id is provided, it will not be possible to refer to this credential once issued nor to deal with errors of duplication. credentialId SHOULD NOT be set by the issuer coordinator if the credential.id property is set, nor should credentialId be used as a replacement for credential.id; rather, credentialId is a means of identifying a credential without the id property being set. The issuer service SHOULD NOT auto-generate a credentialId if one is not supplied by the issuer coordinator, because doing so could create a partitioning error if the result is never received by the client.
```

Comment:
```suggestion
          description: A URI that can be used to identify the credential, that can be used in APIs to refer to the issued verifiable credential. If credentialId is not provided by the issuer coordinator, the issuer service will auto-populate its value from credential.id. If neither credentialId nor credential.id is provided, it will not be possible to refer to this credential once issued nor to deal with errors of duplication. credentialId SHOULD NOT be set by the issuer coordinator if the credential.id property is set, nor should credentialId be used as a replacement for credential.id; rather, credentialId is a means of identifying a credential without the id property (i.e., credential.id) being set. The issuer service SHOULD NOT auto-generate a credentialId if one is not supplied by the issuer coordinator, because doing so could create a partitioning error if the result is never received by the client.
```